app/Http/Livewire/LaravelLogViewer.php

```diff
@@ -55,7 +55,9 @@ final public function getEntriesProperty(): \Illuminate\Contracts\Pagination\Len
         $logString = '';
 
         foreach ($this->logs as $log) {
```

Comment:
Does `foreach ($this->logs ?? [] as $log)` or `foreach ($this->logs ?: [] as $log)` fix the issue? If so, could you try using it instead of the `isset()`?

Reply:
?? and ?: did not work sadly.

Reply:
Darn, let's not worry about this then. Probably best to move it back to how you had originally implemented it.